README.md

```diff
@@ -160,6 +160,15 @@ Endpoint to use for connection to Amazon S3. Leave empty if default S3 endpoint
   * Default: ""
   * Importance: low
 
+``large.message.compression.type``
+The compression type for data stored in blob storage. The default is `none` (i.e. no compression). Valid values are `none`,
+`gzip`, `snappy`, `lz4` and `zstd`.
```

Comment:
Please add a note that this is only used if header usage is configured. Also add it to the config class